pkgs/by-name/tp/tproxy/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "tproxy";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "kevwan";
+    repo = "tproxy";
+    rev = "v${version}";
+    hash = "sha256-d4ZijF3clu00WZQGlurTkGkedurjt9fqfShdjbZWCSI=";
+  };
+
+  vendorHash = "sha256-tnVzX0crDdkRND7Au0CaTdmwLVxVjxU1jxCRutl48S8=";
+
+  meta = with lib; {
+    description = "A cli tool to proxy and analyze TCP connections.";
```

```suggestion
    description = "A cli tool to proxy and analyze TCP connections";
```

No period at the end please.